teachertool/src/teacherTool.css

```diff
@@ -44,6 +44,10 @@
     --high-contrast-hyperlink: #807FFF;
 }
 
+* {
```

Comment:
This is to reset some global styling coming from the target css that was messing up default div layout.